### PR TITLE
jormungandr: delete autocomplete from bragi json

### PR DESCRIPTION
**Pull request by azime**
- delete "addr:" from address id

### CODE REVIEW

File: source/jormungandr/jormungandr/autocomplete/elastic_search.py [hunk 1]
```diff
@@ -275,10 +276,21 @@ def output(self, key, obj):
...
 }
 
+
+class AddressId(fields.Raw):
+    def output(self, key, obj):
+        if not obj:
+            return None
+        geocoding = obj.get('properties', {}).get('geocoding', {})
+        id = geocoding.get('id')
+        if id.startswith("addr:"):
```
Comment (xlqian) [inline source/jormungandr/jormungandr/autocomplete/elastic_search.py]: minor:

```
prefix = "addr:"
if id.startswith(prefix):
     return id[len(prefix):]
```

Comment (azime) [inline source/jormungandr/jormungandr/autocomplete/elastic_search.py]: ok

File: source/jormungandr/jormungandr/autocomplete/elastic_search.py [hunk 2]
```diff
@@ -296,10 +308,20 @@ def format(self, place):
...
+    "places": fields.List(GeocodejsonFeature, attribute='features')
 }
 
 
+class delete_attribute_autocomplete():
+    def __call__(self, f):
+        @wraps(f)
+        def wrapper(*args, **kwargs):
+            objects = f(*args, **kwargs)
+            if 'Autocomplete' in objects:
```
Comment (xlqian) [inline source/jormungandr/jormungandr/autocomplete/elastic_search.py]: you can do 

```
return objects.get('Autocomplete') or objects
```

Comment (azime) [inline source/jormungandr/jormungandr/autocomplete/elastic_search.py]: ok

**Pull request merged**